main.rb

```diff
@@ -1,10 +1,110 @@
-require 'person'
-require 'capitalize'
-require 'trimmer'
-
-person = Person.new(22, 'maximilianus')
-person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+require 'app'
+
+def command
+  puts 'Please  choose an option by enterin a number : '
+  puts ''
+  puts '1- List all books'
+  puts '2- List all people'
+  puts '3- Create a person'
+  puts '4- Create a book'
+  puts '5- Create a rental '
+  puts '6- List all rentals for a given person id'
+  puts '7- Exits'
+end
+
+def block(inp, person_table, app); end
+
+def looping(inp, person_table, book_table, app); end
+
+def main
```

Comment:
- [ ] am not able to create a rental, book and anytime the user lists a record, it exits the app, I think you need to fix these as shown in the record below
![6](https://github.com/francksefu/school_library_oop/assets/77843050/aeda74db-ff12-4512-a6db-a016712ef9ed)